bin/import_gram_vaani.py

```diff
@@ -0,0 +1,302 @@
+    Returns:
+      :obj:`argparse.Namespace`: command line parameters namespace
+    """
+    parser = argparse.ArgumentParser(
+        description="Imports GramVaani data for Deep Speech"
+    )
+    parser.add_argument(
+        "--version",
+        action="version",
+        version="GramVaaniImporter {ver}".format(ver=__version__),
+    )
+    parser.add_argument(
+        "-v",
+        "--verbose",
+        action="store_const",
+        required=False,
+        help="set loglevel to INFO",
+        dest="loglevel",
+        const=logging.INFO,
+    )
+    parser.add_argument(
+        "-vv",
+        "--very-verbose",
+        action="store_const",
+        required=False,
+        help="set loglevel to DEBUG",
+        dest="loglevel",
+        const=logging.DEBUG,
+    )
+    parser.add_argument(
+        "-f",
+        "--file",
+        required=True,
+        help="Path to the GramVaani csv",
+        dest="csv_filename",
+    )
+    parser.add_argument(
+        "-d",
+        "--directory",
+        required=True,
+        help="Directory in which to save the importer GramVaani data",
+        dest="directory",
+    )
+    return parser.parse_args(args)
+
+def setup_logging(level):
+    """Setup basic logging
+    Args:
+      level (int): minimum log level for emitting messages
+    """
+    format = "[%(asctime)s] %(levelname)s:%(name)s:%(message)s"
+    logging.basicConfig(
+        level=level, stream=sys.stdout, format=format, datefmt="%Y-%m-%d %H:%M:%S"
+    )
+
+class GramVaaniCSV:
+    """GramVaaniCSV representing a GramVaani dataset.
+    Args:
+      csv_filename (str): Path to the GramVaani csv
+    Attributes:
+        data (:class:`pandas.DataFrame`): `pandas.DataFrame` Containing the GramVaani csv data
+    """
+
+    def __init__(self, csv_filename):
+        self.data = self._parse_csv(csv_filename)
+
+    def _parse_csv(self, csv_filename):
+        _logger.info("Parsing csv file...%s", os.path.abspath(csv_filename))
+        data = pd.read_csv(
+            os.path.abspath(csv_filename),
+            names=["piece_id","audio_url","transcript_labelled","transcript","labels","content_filename","audio_length","user_id"],
+            usecols=["audio_url","transcript","audio_length"],
+            skiprows=[0],
+            engine="python",
+            encoding="utf-8",
+            quotechar='"',
+            quoting=csv.QUOTE_ALL,
+        )
+        data.dropna(inplace=True)
+        _logger.info("Parsed %d lines csv file." % len(data))
+        return data
+
+class GramVaaniDownloader:
+    """GramVaaniDownloader downloads a GramVaani dataset.
+    Args:
+      gram_vaani_csv (GramVaaniCSV): A GramVaaniCSV representing the data to download
+      directory (str): The path to download the data from
+    Attributes:
+        data (:class:`pandas.DataFrame`): `pandas.DataFrame` Containing the GramVaani csv data
+    """
+
+    def __init__(self, gram_vaani_csv, directory):
```

Comment:
How about a more descriptive parameter name for `directory`?

Reply:
OK, will change the name.